python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py

```diff
@@ -31,7 +31,6 @@ async def deploy_configs_to_cv(configs: list[CVEosConfig], result: DeployToCvRes
     - Upload Configlets and assign to devices.
 
     TODO: See if this can be optimized to check if the configlets are already in place and correct. A hash would have been nice.
-    TODO: Split long configs into multiple configlets for 990KB chunks. Need to figure out how to batch it.
```

Comment:
this seems to be related to the other PR?

Reply:
as mentioned in the PR text, I am cleaning up a few needless TODOs, since this was written during a complete TODO walkthrough.